src/types/request.rs

```diff
@@ -256,25 +256,55 @@ impl PyRequest {
 
     pub fn json(&self, py: Python) -> PyResult<PyObject> {
         match self.body.as_ref(py).downcast::<PyString>() {
-            Ok(python_string) => match serde_json::from_str(python_string.extract()?) {
-                Ok(Value::Object(map)) => {
-                    let dict = PyDict::new(py);
-
-                    for (key, value) in map.iter() {
-                        let py_key = key.to_string().into_py(py);
-                        let py_value = match value {
-                            Value::String(s) => s.as_str().into_py(py),
-                            _ => value.to_string().into_py(py),
-                        };
+            Ok(python_string) => json_string_to_pyobject(py, python_string.extract()?),
+            Err(e) => Err(e.into()),
+        }
+    }
+}
 
-                        dict.set_item(py_key, py_value)?;
-                    }
+///serde_json not impl IntoPy<PyObject>  but orphan principle. so use this function
+fn json_string_to_pyobject(py: Python, json_string: &str) -> PyResult<PyObject> {
+    /// Converts a serde_json::Number to a Python object.
+    fn number_to_pyobject(num: serde_json::Number, py: Python) -> PyResult<PyObject> {
+        match num.as_u64() {
+            Some(u) => Ok(u.into_py(py)),
+            None => match num.as_i64() {
+                Some(i) => Ok(i.into_py(py)),
+                None => match num.as_f64() {
+                    Some(f) => Ok(f.into_py(py)),
+                    None => Err(PyErr::new::<pyo3::exceptions::PyTypeError, _>(
+                        "Invalid number format",
+                    )),
+                },
+            },
+        }
+    }
```

Comment:
how does this handle `-1` ?

Reply:
-1   will  be  python  int(-1).
It wil  handle  by num.as_i64()   case   `N::NegInt(n) => Some(n)`,   it will into to  Python  int .  
In this function .  try parse to  uint   failed , try  parse to  int  failed , try  parse  to  float ,